### ISSUE TITLE
Add clarifying text to 'Enable Connectors' page to reduce confusion

### ISSUE THREAD

**Issue by hwalker** [salesagility/SuiteCRM#1713]
#### Issue

In SuiteCRM Administration (v7.6.5), under Connector Settings / Enable Connectors, its not immediately obvious that you can drag & drop the fields back and forth between "Disabled" and "Enabled".
#### Expected Behavior

The should be text somewhere on this page explaining that the interface is drag & drop
#### Actual Behavior

There currently is no text as described
#### Possible Fix

Suggestion: add text just below the "Save" and "Cancel" buttons (but above the tabs) that says:

"Please drag and drop items to the desired category, and then click 'Save'"
#### Steps to Reproduce

Relative URL:
/index.php?module=Connectors&action=ModifyDisplay
(in Chrome Version 51.0.2704.103 m)
#### Context

This page was very confusing, because typically drag & drop controls don't have the look of a basic list control like this one does.  It's non-intuitive.
#### Your Environment

- SuiteCRM Version used:7.6.5
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):Chrome Version

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add clarifying text to 'Enable Connectors' page to reduce confusion](https://trello.com/c/ojXDU5Y6/89-add-clarifying-text-to-enable-connectors-page-to-reduce-confusion)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
